salt/annotations/config/home-deployuser-annotations-docker-compose.yml

```diff
@@ -12,4 +12,14 @@ services:
         ports:
             - 9000:9000
         restart: always
+    queue_watch:
+        image: "elifesciences/annotations_cli:${IMAGE_TAG}"
+        command: bin/console queue:watch
+        volumes:
+            - /srv/annotations/config.php:/srv/annotations/config.php
```

Comment:
is this a list of maps or a list of strings?

Reply:
It's a list of strings, but the content of the string follows the Docker volume format and hence uses a colon to separate host path and container path:
https://docs.docker.com/engine/admin/volumes/volumes/#choose-the--v-or-mount-flag